docs/hub-tutorials/modules/mauth/mauth-module-tutorial.md

```diff
@@ -0,0 +1,215 @@
+```shell
+gaiad tx bank send $AICA $B --chain-id test-2 --from a --gas 90000 --home test-1 --generate-only | jq '.body.messages[0]' > ./send-raw.json
+
+cat send-raw.json
+{
+  "@type": "/cosmos.bank.v1beta1.MsgSend",
+  "from_address": "cosmos13ys0vw7uhw5c70lrgzz6nw77f95k2pm42rt33areg33k0kltn2zsdjsfvu",
+  "to_address": "cosmos13f43j8hyf63mhz4pkqkcnncpu7fvuxnmh9yxdw",
+  "amount": [
+    {
+      "denom": "stake",
+      "amount": "100"
+    }
+  ]
+}
+```
+step 2: send the generated transaction with `a` singing it:
+```shell
+gaiad tx icamsgauth submit-tx ./send-raw.json --connection-id connection-0 --counterparty-connection-id connection-0 --from a --chain-id test-1 --gas 150000 --home test-1
+```
+
+
+#### Q2: `b` sends the tokens back to `aica`
+
+```shell
+gaiad tx bank send $B $AICA 100stake --home test-2
+```
+
+#### Q3: `a` sends tokens to `b` via IBC
+Create a hermes channel:
+
+```shell
+cp hermes/rly-config.toml ~/.hermes/config.toml
+```
+```shell
+hermes create channel --port-a transfer --port-b transfer test-1 test-2
+```
+
+kill hermes and restart
+```shell
+hermes -c hermes/rly-config.toml start
+```
+
+```shell
+gaiad tx ibc-transfer transfer transfer channel-1 $B 200stake --from a --home test-1
+```
+
+```shell
+gaiad q bank balances $B --home test-2
+balances:
+- amount: "200"
+  denom: ibc/3C3D7B3BE4ECC85A0E5B52A3AEC3B7DFC2AA9CA47C37821E57020D6807043BE9
+- amount: "99999999000"
+  denom: stake
+pagination:
+  next_key: null
+  total: "0"
+```
+
+#### Q4: `b` sends the ibc/stake to `aica`
+```shell
+gaiad tx bank send $B $AICA 200ibc/3C3D7B3BE4ECC85A0E5B52A3AEC3B7DFC2AA9CA47C37821E57020D6807043BE9 --from b --home test-2
+```
+#### Q5: `aica` sends 100ibc/stake to `a`
+step 1: prepare the transaction json file
+```shell
+gaiad tx ibc-transfer transfer transfer channel-1 $A 100ibc/3C3D7B3BE4ECC85A0E5B52A3AEC3B7DFC2AA9CA47C37821E57020D6807043BE9 --from $AICA --home test-2 --generate-only | jq '.body.messages[0]' > send-raw.json 
+
+ cat send-raw.json
+ {
+  "@type": "/ibc.applications.transfer.v1.MsgTransfer",
+  "source_port": "transfer",
+  "source_channel": "channel-1",
+  "token": {
+    "denom": "ibc/3C3D7B3BE4ECC85A0E5B52A3AEC3B7DFC2AA9CA47C37821E57020D6807043BE9",
+    "amount": "100"
+  },
+  "sender": "cosmos13ys0vw7uhw5c70lrgzz6nw77f95k2pm42rt33areg33k0kltn2zsdjsfvu",
+  "receiver": "cosmos1sjww7vhxhe5sfye44fex4fv9telmuakuahk9nh",
+  "timeout_height": {
+    "revision_number": "1",
+    "revision_height": "4130"
+  },
+  "timeout_timestamp": "1641572037493534000"
+}
+```
+step 2: use interchain account to execute the ibc transfer in the json file:
+```shell
+gaiad tx icamsgauth submit-tx send-raw.json --connection-id connection-1 --counterparty-connection-id connection-1 --from a --home test-1 --gas 150000
+```
+The ibc/stake will changed from the denom of `0ibc/3C3D7B3BE` to `stake` when the token is back to a on chain test-1.
+ You might need to kill hermes and restart to receive the tokens.
```

Comment:
I am not sure about `you might need to kill hermes and restart to receive the tokens`  @glnro , when you try, could you please confirm ? Thank you!

Reply:
Sure I'll take a look